src/js/tabs/configuration.js

```diff
@@ -1103,7 +1102,7 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
 
             function save_sensor_alignment() {
                 var next_callback = save_esc_protocol;
-               MSP.send_message(MSPCodes.MSP_SET_SENSOR_ALIGNMENT, mspHelper.crunch(MSPCodes.MSP_SET_SENSOR_ALIGNMENT), false, next_callback);
+                MSP.send_message(MSPCodes.MSP_SET_SENSOR_ALIGNMENT, mspHelper.crunch(MSPCodes.MSP_SET_SENSOR_ALIGNMENT), false, next_callback);
```

Comment:
There is a stray space that's crept in:

```suggestion
               MSP.send_message(MSPCodes.MSP_SET_SENSOR_ALIGNMENT, mspHelper.crunch(MSPCodes.MSP_SET_SENSOR_ALIGNMENT), false, next_callback);
```

Reply:
I can roll this back if you would like but, it was pushed back a line too far before and this change was fixing it to properly align with the line above(at least in my IDE lol)